src/constants/app.constants.js

```diff
@@ -0,0 +1,3 @@
+export const ADD_ONE = 1;
+export const ADD_100 = 100;
```

Comment:
It is customary to name functions from verbs, so now the names are not quite correct:
```suggestion
export const NUMBER_FOR_ADD_ONE = 1;
export const NUMBER_FOR_ADD_100 = 100;
```
Or, if we do not know the context of these values at all and we are writing a very small application - it is better not to create constants for such values.